routes/session.js

```diff
@@ -31,5 +31,25 @@ exports.create = function(req, res) {
 		if (err) return console.err(err);
 
 		console.log(sessionObject);
+		Class.findOne( {_id: req.classId}), function (err, session) {
```

Comment:
Since we're searching for classes here, we can have `class` as a parameter to the function rather than `session`. Then down below we can do `class.sessions.push(...)`